### ISSUE TITLE
Candida esophagitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6072]
**Preferred term label:**
Candida esophagitis

**Synonyms**
Esophageal candidiasis

**Definition (free text, please give PubMed ID)**
Candida infections of the esophagus are considered opportunistic infections and are seen most commonly in immunosuppressed patients, the most common symptoms being dysphagia, odynophagia, and retrosternal pain.
PMID: 30725953 

**Parent term (use hpo.jax.org/app)**
Recurrent candida infections

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 618213

**Gene association**
TGFB1 (OMIM: 190180)

**Comment by pnrobinson** [comment]
added term
